src/api/Metadata.js

```diff
@@ -135,16 +151,22 @@ class Metadata extends File {
      *
      * @param {string} id - file id
      * @param {string} scope - metadata scope
+     * @param {string|void} [instanceId] - metadata instance id
      * @return {Object} array of metadata templates
      */
-    async getTemplates(id: string, scope: string): Promise<Array<MetadataEditorTemplate>> {
+    async getTemplates(id: string, scope: string, instanceId?: string): Promise<Array<MetadataEditorTemplate>> {
+        this.errorCode = ERROR_CODE_FETCH_METADATA_TEMPLATES;
```

Comment:
How does `this.errorCode` get used in this method?

Reply:
Its not used here, its used in the parent Base.js errorHandler()